hack/preserve-bundle-labels.sh

```diff
@@ -65,6 +66,9 @@ sed -i "s|image: mustchange|image: ${KUEUE_OPERATOR_IMAGE}|g" "${CSV_FILE}"
 # Replace Display Name
 sed -i "s|displayName: Kueue Operator|displayName: ${DISPLAY_NAME}|" "${CSV_FILE}"
 
+# Replace Description
+sed -i "s|description: Kueue Operator description. TODO.|description: ${DESCRIPTION}|" "${CSV_FILE}"
```

Comment:
Is there a way to add a description for the Kueue CR as well?

Reply:
I will do this as a follow up.

I think I can use olm.